### PR TITLE
#25 killer

### PR DESCRIPTION
**Pull request by magnetenstad**
Current implementation does not support undo operation.

### CODE REVIEW

**Comment by magnetenstad** [comment]
Closes #25

**Comment by magnetenstad** [comment]
Don't merge this yet.

File: src/types/game.ts [hunk 1]
```diff
@@ -50,31 +54,52 @@ export type Leg = Omit<
...
-  game: Game
-  getCurrentLegScore(): number
+export interface GameState {
+  results: string[]
+  userId: string | null
+  prevUserId: string | null
   getUserResultText(userId: string): string
   getUserDisplayText(userId: string): string
   getSegmentText(segment?: Segment | null): string
+}
```
Comment (GregorMoe) [inline src/types/game.ts:72]: ok!

File: src/types/ts.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export type Prettify<T> = {
+  [K in keyof T]: T[K]
+} & {}
```
Comment (GregorMoe) [inline src/types/ts.ts:3]: personally not how i would do it but could work hmmm

**Review by GregorMoe — APPROVED** [review]
mhmm good!! looks to me :-)

**Pull request merged**